src/components/Message/Message.js

```diff
@@ -5,28 +5,50 @@ const Message = props => {
 	const { username, text } = props;
 	return (
 		<View style={styles.container}>
-			<View>
-				<Text style={styles.username}>{username}</Text>
+			<View style={styles.initialContainer}>
+				<Text style={styles.initials}>{username[0].toUpperCase()}</Text>
```

Comment:
Why is username in an array? I'd consider taking it out of the array before we render it to help with readability.